item-spec/common-metadata.md

```diff
@@ -212,3 +213,61 @@ optical and short-wave IR bands are all 30 meters, but the panchromatic band is
+| self       | *Absolute* URL to the location that the STAC file can be found online, if available.                                | application/json                                     |
+| root       | URL to the root STAC entity ([Catalog](../catalog-spec/README.md) or [Collection](../collection-spec/README.md)).   | application/json                                     |
+| parent     | URL to the parent STAC entity ([Catalog](../catalog-spec/README.md) or [Collection](../collection-spec/README.md)). | application/json                                     |
+| collection | URL to the parent Collection. *Absolute* URLs should be used whenever possible.                                     | application/json                                     |
+| child      | URL to a child STAC entity ([Catalog](../catalog-spec/README.md) or [Collection](../collection-spec/README.md)).    | application/json                                     |
+| item       | URL to a STAC Item.                                                                                                 | application/geo+json (preferred) or application/json |
+
+#### Self relation
+
+The `self` relation is used to link to the STAC entity itself.
+This is particularly useful when in a download package that includes metadata, so that the downstream user can know where the data has come from.
+
+#### Root and parent relation
+
+The `root` and `parent` relations are used to link to the root and parent STAC entity, which is either a [Catalog](../catalog-spec/README.md) or a [Collection](../collection-spec/README.md).
+**ONLY ONE** link with `root` or `parent` relationship and `application/json` type is allowed.
+Other version of the `root` or `parent` link can be added with different `type` field as long as they point the same conceptual entity (e.g. identified by the ID).
+
+#### Collection relation
+
+The referenced Collection is STRONGLY RECOMMENDED to implement the same STAC version as the Collection.
+A link with this `rel` type is *required* for STAC item if the `collection` field in properties is present.
+
+#### Item relation
+
+All Items linked from a Collection MUST refer back to its Collection with the [`collection` relation type](#collection-relation).
+
+### Additional relations
+
+| Type         | Description                                                                                                      | Media Type       |
+| ------------ | ---------------------------------------------------------------------------------------------------------------- | ---------------- |
+| license      | The license URL(s) for the Item SHOULD be specified if the `license` field is **not** a SPDX license identifier. | Any              |
+| derived_from | URL to a STAC Entity that was used as input data in the creation of this Entity.                                 | application/json |
```

Comment:
Maybe we should move the derived_from to the best practices and license back to the common metadata field. But that doesn't need to be part of this PR necessarily.